client/src/pages/Navbar.js

```diff
@@ -142,14 +142,15 @@ function Navbar(props) {
               type="submit"
               variant="contained"
               color="primary"
-              onClick={() => setShowModal(true)}>
+              onClick={() => setShowModal(true)}
+            >
               <AddIcon className={classes.fixRightMargin} />
               <Typography variant="body1" className={classes.fixRightMargin}>
                 Create board
               </Typography>
             </Button>
             <img
```

Comment:
This didn't come from your PR, but you an swap this with a Material UI Avatar component to make the styling easier